tests/integration_test/integration_test.sh

```diff
@@ -398,36 +391,42 @@ ss2_workflow_id=$(docker run --rm -v $script_dir:/app \
 
 printf "\nss2_workflow_id: $ss2_workflow_id"
 
-# 10. Poll for completion
+# 9. Poll for completion
 printf "\n\nAwaiting workflow completion\n"
 
 if [ $use_caas ]; then
-    python $script_dir/await_workflow_completion.py \
-      --workflow_ids $ss2_workflow_id \
-      --workflow_names ss2 \
-      --cromwell_url https://cromwell.caas-dev.broadinstitute.org \
-      --caas_key $lira_dir/kubernetes/caas_key.json \
-      --timeout_minutes 120
+    docker run --rm -v $script_dir:/app \
+        -v $lira_dir/kubernetes/caas_key.json:/etc/lira/caas_key.json \
+        -e WORKFLOW_IDS=$ss2_workflow_id \
+        -e WORKFLOW_NAMES=ss2 \
+        -e CROMWELL_URL=https://cromwell.caas-dev.broadinstitute.org \
+        -e CAAS_KEY=/etc/lira/caas_key.json \
+        -e TIMEOUT_MINUTES=120 \
+        --link lira:lira \
+        broadinstitute/python-requests /app/await_workflow_completion.py
+
 else
-    export cromwell_user=$(docker run -i --rm \
+    export CROMWELL_USER=$(docker run -i --rm \
         -e VAULT_TOKEN=$vault_token \
             broadinstitute/dsde-toolbox \
             vault read -field=cromwell_user secret/dsde/mint/$env/common/htpasswd)
 
-    export cromwell_password=$(docker run -i --rm \
+    export CROMWELL_PASSWORD=$(docker run -i --rm \
         -e VAULT_TOKEN=$vault_token \
         broadinstitute/dsde-toolbox \
         vault read -field=cromwell_password secret/dsde/mint/$env/common/htpasswd)
 
-    python $script_dir/await_workflow_completion.py \
-      --workflow_ids $ss2_workflow_id \
-      --workflow_names ss2 \
-      --cromwell_url https://cromwell.mint-$env.broadinstitute.org \
-      --timeout_minutes 120
+    docker run --rm -v $script_dir:/app \
+        -e WORKFLOW_IDS=$ss2_workflow_id \
+        -e WORKFLOW_NAMES=ss2 \
+        -e CROMWELL_URL=https://cromwell.caas-dev.broadinstitute.org \
+        -e TIMEOUT_MINUTES=120 \
+        --link lira:lira \
+        broadinstitute/python-requests /app/await_workflow_completion.py
```

Comment:
Can you check in the Dockerfile for python-requests to this repo or add a code comment pointing to it if it's elsewhere?